src/main/java/org/apache/commons/imaging/formats/bmp/BmpImageParser.java

```diff
@@ -327,7 +331,7 @@ private BmpImageContents readImageContents(final InputStream is,
         switch (bhi.compression) {
         case BI_RGB:
             if (verbose) {
```

Comment:
Seeing "if verbose" and then "log at the info level" is slightly confusing here. If we want to keep the 'verbose' setting then maybe the code should rename this to 'logInfo', or, should we provide a setting that contains the log level to use when 'verbose' is enabled?

Reply:
Seems to me it would be better to use the log-level alone to control what is being logged.

So the verbose flag would become a log-level instead.

Reply:
You are both correct @garydgregory , @sebbASF .

I tried to remove the flags `DEBUG` or `isDebug()` or `getDebug()` by `LOGGER.isLoggable(Level.FINE)`, but forgot to do the same for `verbose` using the `INFO` level (wonder if we should use FINEST for debug, and FINE for the verbose messages...)
Will just log to info. I believe we can assume that

Reply:
Done in https://github.com/apache/commons-imaging/pull/35/commits/96b0a18e3c0ac435da85a3cf361975e2e77aa7f0

Also removed the `PARAM_KEY_VERBOSE` constant, used to include a param in a map to enable verbose. I initially put a `@deprecated` annotation, then realized this will be 1.0 of imaging, not sanselan, so removing it should be OK.

Reply:
And on

>(wonder if we should use FINEST for debug, and FINE for the verbose messages...)

If you have the chance to run the tests, you will notice the huge amount of verbose messages being printed now. I believe some users could be inundated with log lines (as the INFO is enabled by default? and verbose was disabled by default)